### ISSUE TITLE
import error when using react-flow-renderer v10 on stackblitz

### ISSUE THREAD

**Issue by kafkahw** [wbkd/react-flow#2028]
### Describe the Bug

When building the [overview example](https://reactflow.dev/docs/examples/overview/) on stackblitz using `react-flow-renderer` v10.0.6, I'm seeing this error:
```
Error in /turbo_modules/react-flow-renderer@10.0.6/dist/esm/index-5236d2af.js (8:35)
Cannot use import statement outside a module
```

NOTE: I just copied the code from the react-flow's official site and didn't make any changes.

### Your Example Website or App

https://stackblitz.com/edit/react-ts-44qyxp?file=App.js

### Steps to Reproduce the Bug or Issue

1. go to the example website provided above
2. you'll see the error

### Expected behavior

App should load and display properly without any errors.

### Screenshots or Videos

![react-flow-error](https://user-images.githubusercontent.com/2274728/160534811-d32dbbfc-935b-4044-9691-764b2fab4b67.png)

### Platform

- OS: MacOS
- Browser: Chrome
- Version: 99.0.4844.83

**Comment by moklick** [comment]
React Flow only comes with a ESM build. We will add a UMD build too. Thanks for the report!

**Comment by kafkahw** [comment]
@moklick that's good to know! do you have an estimate on when we can expect that release with UMD build? I'm currently using `CodeSandbox` as a workaround, but would like to build apps on `Stackblitz` as soon as possible.

**Comment by moklick** [comment]
fixed with v10.1.0
